app/exporters/dhis2_exporter.rb

```diff
@@ -33,8 +33,6 @@ def export
       repository = Reports::Repository.new(facility.region, periods: range)
 
       range.each do |period|
```

Comment:
I think this code would be slightly clearer if we renamed `period` to `month` here and in `reporting_period` below. If I understand the code, it would break if we used quarters.